chrome/content/zotero/zoteroPane.xul

```diff
@@ -517,6 +517,7 @@
 									class="treecol-image"
 									label="&zotero.tabs.attachments.label;"
 									src="chrome://zotero/skin/attach-small.png"
+									fixed="true"
 									zotero-persist="width ordinal hidden sortActive sortDirection"/>
```

Comment:
Also need to remove "width" from zotero-persist. If someone has that column at non-standard width, they would be stuck with it.